gridscale/resource_gridscale_server.go

```diff
@@ -68,9 +70,11 @@ func resourceGridscaleServer() *schema.Resource {
 				},
 			},
 			"storage": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
 				Optional: true,
 				MaxItems: 8,
+				Description: `A list of storages attached to the server. The storage in the 1st order of the list is always the boot storage.
+In order to replace the boot storage, the new boot storage must be placed in the 1st order of the list.`,
 				Elem: &schema.Resource{
```

Comment:
Consider rewording comment
- "The storage in the 1st order of the list is always the boot storage."
+ "The first storage in the list is always set as the boot storage of the server."